src/App.tsx

```diff
@@ -1,24 +1,215 @@
+/* eslint-disable no-restricted-syntax */
 /* eslint-disable max-len */
+/* eslint-disable no-console */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  deleteTodo, getTodos, patchTodo, postTodo,
+} from './api/todos';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { ErrorMessages, Filter } from './types/enum';
+import { TodoContext } from './TodoContext';
+import { State, Todo } from './types/interfaces';
+import { TodoErrorNotification } from './components/TodoErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 82;
 
 export const App: React.FC = () => {
+  const [state, setState] = useState<State>({
+    todos: [],
+    filter: Filter.All,
+  });
+
+  const [error, setError] = useState<string | null>(null);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState<number[]>([]);
```

Comment:
`isLoading` sounds like it has boolean value but you have an array

please think about another name